nodeup/pkg/model/miscutils.go

```diff
@@ -51,7 +51,7 @@ func (b *MiscUtilsBuilder) Build(c *fi.ModelBuilderContext) error {
 			packages = append(packages, "apt-transport-https")
 
 			// TODO: Do we really need python-apt?
-			if b.Distribution.IsUbuntu() && b.Distribution.Version() >= 20.10 {
+			if (b.Distribution.IsUbuntu() && b.Distribution.Version() >= 20.10) || (!b.Distribution.IsUbuntu() && b.Distribution.Version() >= 11) {
```

Comment:
We probably should have `IsDebian` rather than `!IsUbuntu`, just because it's more self-documenting, but ... that's just a nit.

Reply:
Yes, probably we should look to add this in a future PR.